### PR TITLE
Modifying boto3 examples

### PR DESCRIPTION
**Pull request by faithuniterh**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by faithuniterh** [comment]
@yuvalif This is the draft merge request of the work I have began on the boto3 python examples.

File: examples/boto3/list_unordered.py [hunk 1]
```diff
@@ -22,4 +22,4 @@
 
 # getting an unordered list of objects is an extension to AWS S3 API
 
-print(client.list_objects(Bucket=bucketname, AllowUnordered=True))
+print(client.list_objects(Bucket=bucketname))
```
Comment (yuvalif) [inline examples/boto3/list_unordered.py:25]: we want to demonstrate our extension that allow for unordered listing

File: examples/boto3/topic_with_endpoint.py [hunk 2]
```diff
@@ -34,8 +32,6 @@
 # radosgw-admin realm zonegroup list
 
 # this is standard AWS services call, using custom attributes to add AMQP endpoint information to the topic
+attributes = {"push-endpoint": "amqp://localhost:5672", "amqp-exchange": "ex1", "amqp-ack-level": "broker"}
 
-endpoint_args = 'push-endpoint=amqp://127.0.0.1:5672&amqp-exchange=ex1&amqp-ack-level=broker'
-attributes = {nvp[0] : nvp[1] for nvp in urlparse.parse_qsl(endpoint_args, keep_blank_values=True)}
-
-print(client.create_topic(Name=topic_name, Attributes=attributes))
+print((client.create_topic(Name=topic_name, Attributes=attributes)))
```
Comment (yuvalif) [inline examples/boto3/topic_with_endpoint.py:37]:
```suggestion
print(client.create_topic(Name=topic_name, Attributes=attributes))
```

Comment (faithuniterh) [reply examples/boto3/topic_with_endpoint.py:37]: Oops! Sorry for this Yuval.

File: examples/boto3/list_unordered.py (hunk 1 above)
Comment (faithuniterh) [reply examples/boto3/list_unordered.py:25]: I was testing it on AWS Cli version 1 which does not have the AllowUnordered=True option. I will make the necessary changes.

File: examples/boto3/get_notification.py [hunk 3]
```diff
@@ -3,14 +3,12 @@
 import boto3
 import sys
 
-if len(sys.argv) != 3:
-    print('Usage: ' + sys.argv[0] + ' <bucket> <notification>')
+if len(sys.argv) != 2:
+    print('Usage: ' + sys.argv[0] + ' <bucket>')
```
[the remark this answers is not in the document]
Comment (faithuniterh) [reply examples/boto3/get_notification.py:7]: Thank you for having a look :) .I thought this was to get the all notifications that do exist on the bucket. I will make the changes as required.

File: examples/boto3/topic_with_endpoint.py [hunk 4]
```diff
@@ -15,7 +13,7 @@
     topic_name = sys.argv[1]
     region_name = ""
```
Comment (yuvalif) [inline examples/boto3/topic_with_endpoint.py:14]: `region_name` is not used. please remove

Comment (faithuniterh) [reply examples/boto3/topic_with_endpoint.py:14]: Thanks for this.

**Review by yuvalif — APPROVED** [review]

**Comment by yuvalif** [comment]
jenkins test make check

**Pull request merged**